VisualStudio.gitignore

```diff
@@ -80,6 +80,8 @@ ipch/
 *.opensdf
 *.sdf
 *.cachefile
+*.VC.db
+*.VC.VC.opendb
```

Comment:
Apologies for the delay in getting back to this:

> And temporary file with *.VC.VC.opendb extension (yes, two subsequent VC) is used for new DB engine when solution is opened. 

This should be covered by `*.opendb` on line 79, right?